utils/src/main/kotlin/ArchiveUtils.kt

```diff
@@ -112,6 +118,35 @@ fun File.unpack7Zip(targetDirectory: File) {
  */
 fun File.unpackZip(targetDirectory: File) = ZipFile(this).unpack(targetDirectory)
 
+/**
+ * A list with file names that are expected to be contained in a Debian package archive file.
+ */
+private val DEBIAN_ARCHIVE_CONTENT = listOf("data.tar.xz", "control.tar.xz")
+
+/**
+ * Unpack the [File] assuming it is a Debian archive. A Debian archive is an ar archive that contains itself two tar
+ * files with metadata and the actual content of the package. The top-level archive is deflated into a temporary
+ * directory. Then the tar files are extracted to the provided [targetDirectory].
+ */
+fun File.unpackDeb(targetDirectory: File) {
```

Comment:
Should this be made private?

Reply:
I would say yes, but all the other format-specific `unpack()` functions are public as well.

Reply:
IMO this should stay publish like the other functions to allow unpacking of concrete formats independent of the file extension.

Reply:
Ok with me.